packages/table/src/_docs.scss

```diff
@@ -128,17 +128,54 @@ Loading states
 
 When fetching or updating data, it may be desirable to show a loading state. The Table components
 provide fine-grain loading control of loading row headers, column headers, or individual cells.
+Several table components expose a `loading` or `loadingOptions` prop, but loading-related rendering
+is computed with components lower in the hierarchy taking priority. For example, a cell with
+`loading` set to `false` will never render its loading state even if the `Column` component to which
+it belongs has a `loadingOptions` value of `[ ColumnLoadingOption.CELLS ]`. The following examples
+display a table of the largest potentially hazardous asteroid (based on absolute magnitude)
+discovered in a given year.
+
+Weight: 4
+
+Styleguide components.table-js.loading
+*/
+
+/*
+Table
+
+`Table` expose a `loadingOptions` prop that allows you to control the loading state behavior of all
```

Comment:
no you're not, it's very much singular here. if it's plural then add an `s` somewhere.